jdaviz/configs/default/plugins/model_fitting/fitting_backend.py

```diff
@@ -104,7 +104,8 @@ def _fit_1D(initial_model, spectrum, run_fitter, window=None):
             weights = 'unc'
         else:
             weights = None
-        output_model = fit_lines(spectrum, initial_model, weights=weights, window=window)
+        output_model = fit_lines(spectrum, initial_model, weights=weights,
+                                 filter_non_finite=filter_non_finite, window=window)
```

Comment:
Is there any use in making `filter_non_finite` an optional keyword in a hidden function that user has no direct access to?

Reply:
`fit_lines` didn't accept `**kwargs` before, right?  So we need the upstream PR to be merged, released, and pinned here before we can merge this PR?

Reply:
@pllim  i added filter_non_finite to 'fit_lines' which users have access to in specutils, its set to false by default there so I set to to true by default here.

@kecnry fit_lines does accept **kwargs, so i think this should work

Reply:
`fit_lines` did accept **kwargs, so we can merge this here, it just won't do anything yet.

Reply:
It looks like it passes kwargs through to astropy actually, so the specutils PR just really adds the documentation explicitly for this option?  Maybe this will work without specutils changes at all then?

Reply:
Actually, looking at the test failures on the `specutils` PR, this might actually break things without the proper handling in `specutils`.